playground.go

```diff
@@ -0,0 +1,345 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/appwrite/sdk-for-go/appwrite"
+	"github.com/appwrite/sdk-for-go/client"
+	"github.com/appwrite/sdk-for-go/databases"
+	"github.com/appwrite/sdk-for-go/functions"
+	"github.com/appwrite/sdk-for-go/id"
+	"github.com/appwrite/sdk-for-go/permission"
+	"github.com/appwrite/sdk-for-go/storage"
+	"github.com/appwrite/sdk-for-go/users"
+)
+
+var appwrite_project = "<PROJECT_ID>"
+var appwrite_api_key = "<API_KEY>"
+
+var (
+	appwriteClient   client.Client
+	appwriteDatabase *databases.Databases
+	appwriteFunction *functions.Functions
+	appwriteUsers    *users.Users
+	appwriteStorage  *storage.Storage
+	database_id      string
+	collection_id    string
+	document_id      string
+	user_id          string
+	bucket_id        string
+	file_id          string
+	function_id      string
+)
+
+func main() {
+	appwriteClient = appwrite.NewClient(
+		appwrite.WithProject(appwrite_project),
+		appwrite.WithKey(appwrite_api_key),
+	)
+
+	appwriteDatabase = appwrite.NewDatabases(appwriteClient)
+	appwriteFunction = appwrite.NewFunctions(appwriteClient)
+	appwriteUsers = appwrite.NewUsers(appwriteClient)
+	appwriteStorage = appwrite.NewStorage(appwriteClient)
+
+	CreateUser()
+	ListUsers()
+	DeleteUsers()
+
+	CreateDatabase()
+	DeleteDatabase()
+
+	CreateCollection()
+	ListCollection()
+	DeleteCollection()
+
+	CreateDocument()
+	ListDocuments()
+	DeleteDocument()
+
+	CreateBucket()
+	ListBuckets()
+	DeleteBucket()
+
+	// UploadFile() TODO: Fix how we send content range etc in SDK Go
+	ListFiles()
+
+	CreateFunction()
+	ListFunctions()
+	DeleteFunction()
+
+	fmt.Println("Successfully ran playground!")
+}
+
+func CreateUser() {
+	fmt.Println("Running Create user API")
+
+	user, _ := appwriteUsers.Create(
+		id.Unique(),
+		appwriteUsers.WithCreateEmail("test@example.com"),
+		appwriteUsers.WithCreatePhone("+12065550100"),
+		appwriteUsers.WithCreatePassword("test1234"),
+		appwriteUsers.WithCreateName("test"),
+	)
+
+	user_id = user.Id
+
+	fmt.Println(user)
+}
+
+func ListUsers() {
+	fmt.Println("Running List User API")
+
+	users, _ := appwriteUsers.List()
+
+	fmt.Println(users)
```

Comment:
The output of printing the result like this is not really readable. How about we format using a helper like:

```go
func print(data interface{}) {
	b, _ := json.MarshalIndent(data, "", "  ")
	fmt.Println(string(b))
}
```